lib/src/main/java/com/diffplug/spotless/generic/LicenseHeaderStep.java

```diff
@@ -335,7 +335,7 @@ private String setLicenseHeaderYearsFromGitHistory(String raw, File file) throws
 		}
 
 		private static String parseYear(String cmd, File file) throws IOException {
-			String fullCmd = cmd + " " + file.getAbsolutePath();
+			String fullCmd = cmd + " -- " + file.getAbsolutePath();
```

Comment:
Unsure if it's best to assume the `cmd` supports this, or whether it's better to hoist it up into the caller's strings for each command. Let me know...

Reply:
LGTM, thanks.